documentation/source/platform_support.rst

```diff
@@ -0,0 +1,108 @@
+.. _platform-support:
+
+****************
+Platform Support
+****************
+
+cocotb supports Windows, Linux, and macOS, and a wide range of Python versions.
+This page describes which platform combinations we test and support.
+
+The :ref:`platform-support-policy` discusses the underlying policy.
+
+.. note::
+
+  In many cases cocotb will work on platforms not listed on this page.
+  However, we encourage less experienced users to stick with one of the supported options for a good user experience.
+
+  Please reach out to the cocotb team if you would like to see support for a different platform, and are willing to contribute to maintaining it.
+
+Supported Python Versions
+=========================
+
+The following versions of Python (CPython), and all associated patch releases (e.g. 3.8.4), are supported by cocotb.
+
+* Python 3.6
+* Python 3.7
+* Python 3.8
+* Python 3.9
+* Python 3.10
+
+Supported Linux Distributions and Versions
+==========================================
+
+Linux distributions typically ship a default version of Python called "system Python".
+This version of Python can be used with cocotb unless noted otherwise.
```

Comment:
I had something of that sort in the Operating System section before, and removed it since it was a bit redundant with the note at the top.

But since you stumbled across it as well I added some text back to this section. PTAL.